### ISSUE TITLE
it works but it sometimes uploads

### ISSUE THREAD

**Issue by Lance2006** [theabbie/Youtube-Uploader-Bot#17]
Hey! so the code works but sometimes it will upload and sometimes it won't upload and I already got the videos source and everything but everytime i press the run button it runs but i check my youtube studio and the video isn't there. Can you help me and if i am doing something wrong? Is there a limited times I can run the code or is it like a daily youtube upload limit?

**Comment by theabbie** [comment]
@Lance2006 it could be youtube rate limit, it has limit of 4-5 videos per day, it could also be that you're uploading very small videos which could be blocked by youtube or maybe you uploaded same video. Don't know the issue but these are the possibilities.

**Comment by Lance2006** [comment]
Ok. So would i have to get the bot to upload bigger videos or would i have to upload different videos? I could try to upload longer and bigger videos.

**Comment by theabbie** [comment]
@Lance2006 try longer videos and add longer titles and description for videos

**Comment by Lance2006** [comment]
Ok. I will let you know if it worked. Thanks!

**Comment by Lance2006** [comment]
I think that it is the YouTube rate limit because I tried longer videos and longer titles and It still didn't work. Is there a way to bypass the YouTube rate limit?

**Comment by theabbie** [comment]
@Lance2006 create multiple apps on Apps Script, each has seperate limit, so, you can upload more videos.

**Comment by Lance2006** [comment]
Is there a code where you can upload without any limit? Like in python or apps script?

**Comment by theabbie** [comment]
@Lance2006 nope, youtube will have a limit regardless of tool used, you can upload maximum 100 videos daily even from the website.

**Comment by Lance2006** [comment]
That is a lot of videos you can upload. I wonder if there is a code where you can upload from google drive or something without using a video source?

**Comment by theabbie** [comment]
@Lance2006 search for "drive to youtube" tools on Google, there are many tools, most are either paid or don't work or are unsafe.

**Comment by Lance2006** [comment]
I found this: https://imclient.herokuapp.com/youtubeupload/?state=%7B%22ids%22:%5B%221cTZqhChwhgKt5QJnI8sD5lFIWowPQ4rs%22%5D,%22action%22:%22open%22,%22userId%22:%22117260531531379041081%22,%22resourceKeys%22:%7B%7D%7D

Do you think it will work?

**Comment by theabbie** [comment]
@Lance2006 Might work but could be unsafe, try on another youtube account, though I believe youtube limit applies to all apps so it's unlikely to work any better.
